### PR TITLE
feat(preset): include applicationinsights-dotnet as monorepo

### LINKED ISSUE #16680: Add monorepo for application insights
### What would you like Renovate to be able to do?

The monorepo grouping should include application insights (https://github.com/microsoft/ApplicationInsights-dotnet ).

Application Insights is a standard component from Microsoft, included in many projects. The SDK is seperated into individual nuget packages, which are currently not grouped together.

### If you have any ideas on how this should be implemented, please tell us here.

Add monorepo for https://github.com/microsoft/ApplicationInsights-dotnet .

I could try to submit a PR for this, but could use a pointer on how to do this.

### Is this a feature you are interested in implementing yourself?

Maybe

### PR DESCRIPTION
**Pull request by aKzenT**
## Changes
See #16680 . The PR adds a new monorepo for applicationinsights-dotnet.

## Context
Closes #16680 

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by zharinov — DISMISSED** [review]

**Review by rarkins — DISMISSED** [review]

File: lib/config/presets/internal/monorepo.ts [hunk 1]
```diff
@@ -17,6 +17,8 @@ const repoGroups = {
   'angular-eslint': 'https://github.com/angular-eslint/angular-eslint',
   angularfire: 'https://github.com/angular/angularfire',
   angularjs: 'https://github.com/angular/angular.js',
+  'applicationinsights-dotnet':
```
Comment (JamieMagee) [inline lib/config/presets/internal/monorepo.ts:20]: Can you sort alphabetically?

Comment (aKzenT) [reply lib/config/presets/internal/monorepo.ts]: Fixed. Thanks.

File: lib/config/presets/internal/monorepo.ts [hunk 2]
```diff
@@ -19,6 +19,8 @@ const repoGroups = {
   angularjs: 'https://github.com/angular/angular.js',
   'apollo-server': 'https://github.com/apollographql/apollo-server',
   apolloclient: 'https://github.com/apollographql/apollo-client',
+  'applicationinsights-dotnet':
```
Comment (viceice) [inline lib/config/presets/internal/monorepo.ts:22]: i think we should use `dotnet` as prefix, as we've already a lot monorepos with that schema

**Review by viceice — CHANGES_REQUESTED** [review]

File: lib/config/presets/internal/monorepo.ts (hunk 2 above)
Comment (astellingwerf) [reply lib/config/presets/internal/monorepo.ts:22]: > i think we should use `dotnet` as prefix, as we've already a lot monorepos with that schema

There are two items that are prefixed with `dotnet`, and two are suffixed.
Ruby is only suffixed (but just once).
Javascript also, only just suffixed.

Comment (viceice) [reply lib/config/presets/internal/monorepo.ts:22]: 🤔 Ok, we should unify those, probably always use suffix.

It should be possible to do it without breaking with using `migratePresets` feature.

@HonkingGoose Can you please open an issue to trsack that?

Comment (HonkingGoose) [reply lib/config/presets/internal/monorepo.ts:22]: > @HonkingGoose Can you please open an issue to track that?

Done, see:

- #16766

Comment (aKzenT) [reply lib/config/presets/internal/monorepo.ts:22]: As the plan seems to be using a suffix in the future and this PR already uses a suffix, I marked this as resolved.

**Review by JamieMagee — APPROVED** [review]

**Review by viceice — APPROVED** [review]

**Pull request merged**